homeassistant/components/fronius/config_flow.py

```diff
@@ -56,10 +56,7 @@ async def validate_host(
         _LOGGER.debug(err)
         raise CannotConnect from err
     except StopIteration as err:
-        raise CannotConnect(
-            translation_domain=DOMAIN,
-            translation_key="no_supported_device_found",
-        ) from err
+        raise CannotConnect("No supported Fronius SolarNet device found.") from err
```

Comment:
```suggestion
        raise CannotConnect from err
```
This text is never be used anyway.

Reply:
(If you want to use it add it to the `cannot_connect` in `strings.json`)

Reply:
Right. I still think it's fine to have it as kind of code comment / annotation. Either for future me or for other contributors wondering what's going on.

Reply:
> why are we not aborting

The idea was to return to the user flow so a user can correct possible typos in `host`.

Reply:
I combined both exception cases and removed the extra test for this branch. It is *very* unlikely that there isn't an inverter found in an inverters API response anyway 🙃